src/pretix/presale/templates/pretixpresale/event/fragment_product_list.html

```diff
@@ -17,7 +17,7 @@ <h3 id="category-{{ tup.0.id }}">{{ tup.0.name }}</h3>
             {% if item.has_variations %}
                 <article aria-labelledby="item-{{ item.pk }}-legend"{% if item.description %} aria-describedby="item-{{ item.pk }}-description"{% endif %} class="item-with-variations{% if event.settings.show_variations_expanded %} details-open{% endif %}" id="item-{{ item.pk }}">
                     <div class="row product-row headline">
-                        <div class="col-md-8 col-xs-12">
```

Comment:
Do we need the same changes from fragment_product_list.html in voucher.html as well? Probably.

Reply:
Good catch, I fixed the indent for description, etc. and added `col-sm-` classes to voucher.html and fragment_availability.html. I also fixed an alignment issue when free price input was used.